codegens/golang/test/unit/convert.test.js

```diff
@@ -125,7 +125,7 @@ describe('Golang convert function', function () {
                     options = {
                         indentCount: 1,
                         indentType: 'tab',
-                        requestTimeout: 2000,
+                        requestTimeout: 5000,
```

Comment:
why has this changed?

Reply:
Previously it wasn't used in the code properly as it accepts input in seconds and was used directly . After handling the timeout, some requests are getting bailed out. Hence changed to 5 seconds.